app/sidekiq/lighthouse/submit_benefits_intake_claim.rb

```diff
@@ -46,15 +45,23 @@ def perform(saved_claim_id)
       raise BenefitsIntakeClaimError, response.body unless response.success?
 
       Rails.logger.info('Lighthouse::SubmitBenefitsIntakeClaim succeeded', generate_log_details)
+      StatsD.increment("#{STATSD_KEY_PREFIX}.success")
```

Comment:
@TaiWilkin This is going to help me after this gets merged to replace these StatD with the new class I'm building `Burials::Monitor`